src/pocketmine/command/SimpleCommandMap.php

```diff
@@ -89,6 +90,8 @@ private function setDefaultCommands(){
 			new BanCommand("ban"),
 			new BanIpCommand("ban-ip"),
 			new BanListCommand("banlist"),
+			new DayLockCommand("alwaysday"),
+			new DayLockCommand("daylock"),
```

Comment:
One of these should be registered as an alias instead of duplicating the command.